pkg/ddevapp/config.go

```diff
@@ -1385,28 +1387,37 @@ func WriteImageDockerfile(fullpath string, contents []byte) error {
 func HasAllowedLocation(app *DdevApp) error {
 	homeDir, _ := os.UserHomeDir()
 	if app.AppRoot == homeDir || app.AppRoot == filepath.Dir(globalconfig.GetGlobalDdevDir()) {
-		return fmt.Errorf("'ddev config' is not allowed in your home directory (%v)", app.AppRoot)
+		return fmt.Errorf("a project is not allowed in your home directory (%v)", app.AppRoot)
 	}
 	rel, err := filepath.Rel(app.AppRoot, homeDir)
 	if err == nil && !strings.HasPrefix(rel, "..") {
-		return fmt.Errorf("'ddev config' is not allowed in the parent directory of your home directory (%v)", app.AppRoot)
+		return fmt.Errorf("a project is not allowed in the parent directory of your home directory (%v)", app.AppRoot)
 	}
 	rel, err = filepath.Rel(globalconfig.GetGlobalDdevDir(), app.AppRoot)
 	if err == nil && !strings.HasPrefix(rel, "..") {
-		return fmt.Errorf("'ddev config' is not allowed in your global config directory (%v)", app.AppRoot)
+		return fmt.Errorf("a project is not allowed in your global config directory (%v)", app.AppRoot)
 	}
 	if fileutil.FileExists(filepath.Join(app.AppRoot, "cmd/ddev/main.go")) && fileutil.FileExists(filepath.Join(app.AppRoot, "cmd/ddev_gen_autocomplete/ddev_gen_autocomplete.go")) {
-		return fmt.Errorf("'ddev config' is not allowed in the directory used for DDEV development (%v)", app.AppRoot)
-	}
-	projectList := globalconfig.GetGlobalProjectList()
+		return fmt.Errorf("a project is not allowed in the directory used for DDEV development (%v)", app.AppRoot)
```

Comment:
Nit: I don't think this error message is obvious, but only we should be hitting it.

```suggestion
		return fmt.Errorf("a project cannot be created in the DDEV source code (%v)", app.AppRoot)
```